src/commands/import.ts

```diff
@@ -1120,19 +1144,58 @@ command
 
       logger.info(`Created ${customFieldsToCreate.length} custom field(s)`);
 
-      logger.info('Checking if "Status" field is configured correctly...');
+      // At the time of writing this, the GraphQL mutation 'updateProjectV2Field' is only available on
+      // GitHub.com
+      const shouldConfigureStatusField = typeof gitHubEnterpriseServerVersion === 'undefined'
+      if (shouldConfigureStatusField) {
+        const sourceProjectStatusField = sourceProject.fields.nodes.find(field => field.name == "Status")!;
+        const sourceProjectStatusFieldOptions = sourceProjectStatusField.options!;
+
+        logger.info(`Configuring "Status" field: ${JSON.stringify(sourceProjectStatusFieldOptions)}`);
 
-      const { sourceProjectStatusFieldId, targetProjectStatusFieldId, mappings } =
-        await promptUntilStatusFieldsCorrelate({
+        const targetProjectStatusField = await getProjectStatusField({
           octokit,
-          sourceProject,
-          targetProjectId,
-          targetProjectUrl,
+          projectId: targetProjectId,
         });
-      sourceToTargetCustomFieldMappings.set(sourceProjectStatusFieldId, {
-        targetId: targetProjectStatusFieldId,
-        optionMappings: mappings,
-      });
+
+        const targetProjectStatusFieldOptions = await updateProjectStatusField(
+          {
+            octokit,
+            statusFieldId: targetProjectStatusField.id,
+            options: sourceProjectStatusFieldOptions.map(option => ({
+              name: option.name,
+              color: option.color,
+              description: option.description,
+            } as SelectOption))
+          }
+        )
+
+        const mappings = correlateCustomFieldOptions(
+          sourceProjectStatusField.options as Array<{ id: string; name: string }>,
```

Comment:
We can just use a `!` to let the TypeScript compiler know that this is defined, and then it knows this is the right type:

```suggestion
          sourceProjectStatusField.options!,
```